user/urls.py

```diff
@@ -1,10 +1,14 @@
 from django.urls import path
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView, TokenVerifyView
+
 from user.views import CreateUserView, CreateTokenView, ManageUserView
 
 app_name = "user"
 
 urlpatterns = [
     path("register/", CreateUserView.as_view(), name="create"),
-    path("login/", CreateTokenView.as_view(), name="login"),
+    path("token/", TokenObtainPairView.as_view(), name="token"),
```

Comment:
Ensure that the `CreateTokenView` is correctly implemented in the `user.views` module, as it is referenced here for token creation.